### PR TITLE
Remove astropy_helpers submodule

### LINKED ISSUE #660: Remove astropy_helpers
We've come to the conclusion that we've outgrown the functionality the submodule provides and we never really used its facilities for Cython that much.

### PR DESCRIPTION
**Pull request by StanczakDominik**
Closes #660

### CODE REVIEW

**Comment by StanczakDominik** [comment]
- [ ] `plasmapy.test` is broken right now

**Comment by StanczakDominik** [comment]
It seems that `import plasmapy; plasmapy.test()` works, while `python setup.py test` does not. But I'd honestly much rather just deprecate the latter and use tox instead.

File: ah_bootstrap.py [hunk 1]
```diff
@@ -1,1010 +0,0 @@
-"""
```
Comment (namurphy) [inline ah_bootstrap.py]: I'm going to miss this file, even though I had no idea what it did.

**Comment by StanczakDominik** [comment]
Thanks!

**Pull request merged**